ceph-api-nightly/config/definitions/ceph-api-nightly.yml

```
@@ -99,6 +99,6 @@
           recipients: ceph-qa@ceph.io
 
       - archive:
-          artifacts: 'build/out/mgr.*.log'
+          artifacts: 'build/out/mgr.*.log, build/run/1/out/mgr.*.log, build/run/2/out/mgr.*.log'
```

Recently there was an issue in an API Jenkins job where the RGW daemon crashed, so I thought it would be useful to store the RGW logs too. So what about:
```suggestion
          artifacts: 'build/out/mgr.*.log, build/run/1/out/*.log, build/run/2/out/*.log'
```

---

done!